app.js

```diff
@@ -0,0 +1,17 @@
+const navLinks = document.getElementById('menu-links');
+const hamburger = document.getElementById('hamburger');
+const maincontent = document.getElementById('maincontent');
+const iconCancel = document.querySelector('#iconCancel');
+
+hamburger.addEventListener('click', (() => {
+  navLinks.style.visibility = 'visible';
+  hamburger.style.display = 'none';
+  maincontent.classList.add('blur');
+}));
+
+iconCancel.addEventListener('click', (() => {
+  navLinks.style.visibility = 'hidden';
+  hamburger.style.display = 'block';
+  hamburger.style.position = 'absolute';
+  maincontent.classList.remove('blur');
+}));
```

Comment:
- [ ] Good job implementing the changes suggested by the reviewer. The mobile menu is appearing on clicking the hamburger icon 🥇💪 But there's one more step to go 🚀 Please make sure that when user clicks any of the mobile menu options, the mobile menu disappears. Right now, the mobile menu is not disappearing on clicking the menu options ⚒ Please note that this is a part of the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md). Following is the quote for the related requirement 👇 I know you can fix this 🎯

> _When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears._

https://user-images.githubusercontent.com/106119105/223389997-a4d9520b-d75b-4bd4-abb7-9dd596daecd0.mp4